### DISCUSSION TITLE
facing a weird problem

### DISCUSSION THREAD

**Discussion by Alone0316** [ROM-builders/temporary#13851] [General]
https://cirrus-ci.com/task/6345045010808832 can you help with this? everything was fine before but now suddenly started getting this,  @JaswantTeja  is also facing similar problem in AOSPA and i didn't change anything and i had a talk with him in push group and he is also claiming that he didn't change anything in trees, he and me both suddenly started facing this problem everything was fine before, can you look into it?  https://cirrus-ci.com/task/5448206866710528 link to his build

**Comment by JaswantTeja** [comment]
@Apon77 I Too Facing This in AOSPA !! and the bulid link is up.

**Reply by Apon77** [reply]
> @Apon77 I Too Facing This in AOSPA !! and the bulid link is up. @Alone0316 

https://github.com/AOSPA/manifest/blob/sapphire/aospa.xml#L19-L23

**Comment by Apon77** [comment]
Your local manifest has problem cloning Soong repo. It must have linkfile line like this 

https://github.com/PixelOS-Pixelish/manifest/blob/64608442a61d3f32c42add9dff06cfe9c24c0386/snippets/pixel.xml#L72-L75

**Comment by JaswantTeja** [comment]
@Apon77 Thanks Sir. Acutally Didn't Noticed About The Linkfile In Manifest.
